client/components/pagination/docs/example.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Pagination from 'components/pagination';
+
+class PaginationExample extends Component {
+	constructor() {
+		super();
+		this.state = {
+			page: 1,
+		};
+	}
+
+	updatePage = ( page ) => {
```

Comment:
In arrow functions, parentheses are optional when there's only one parameter.
`updatePage = page => {}`